apps/governance-e2e/src/support/common.functions.ts

```diff
@@ -0,0 +1,86 @@
+const tokenDropDown = 'state-trigger';
+const txTimeout = Cypress.env('txTimeout');
+
+export enum navigation {
+  section = 'nav',
+  vesting = '[href="/token/redeem"]',
+  validators = '[href="/validators"]',
+  rewards = '[href="/rewards"]',
+  withdraw = '[href="/token/withdraw"]',
+  proposals = '[href="/proposals"]',
+  pageSpinner = '[data-testid="splash-loader"]',
+  supply = '[href="/token/tranches"]',
+  token = '[href="/token"]',
+}
+
+export function convertTokenValueToNumber(subject: string) {
```

Comment:
Suggestion (non-blocking): it'd be nice to have unit tests for the functions in this whole directory for an extra layer of testing goodness!

Reply:
I agree 100%. Will look into adding some more unit tests in the future.